test/programs/namespace-deep-1/schema.json

```diff
@@ -55,26 +56,6 @@
             "type": "object"
         }
     },
-    "properties": {
-        "nest": {
-            "$ref": "#/definitions/RootNamespace.Def"
-        },
-        "prev": {
-            "$ref": "#/definitions/RootNamespace.Def"
-        },
-        "propA": {
-            "$ref": "#/definitions/RootNamespace.SubNamespace.HelperA"
-        },
-        "propB": {
-            "$ref": "#/definitions/RootNamespace.SubNamespace.HelperB"
-        }
-    },
-    "required": [
-        "nest",
-        "prev",
-        "propA",
-        "propB"
-    ],
     "type": "object"
```

Comment:
Should the type still be here? We already have a $ref property so I think this is wrong.